sonar/src/main/java/com/rbc/sonar/Main.java

```diff
@@ -5,7 +5,9 @@
  */
 public class Main {
 
+    private static final String PASSWORD = "mypassword";
+
     public static void main(String[] args) {
-        
+        System.out.println(PASSWORD);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9089/coding_rules#rule_key=squid%3AS106)